src/backend/joanie/tests/core/test_api_organizations_courses.py

```diff
@@ -0,0 +1,202 @@
+"""
+Test suite for Organization's courses API endpoint.
+"""
+
+from joanie.core import factories
+from joanie.tests.base import BaseAPITestCase
+
+
+class OrganizationCourseApiTest(BaseAPITestCase):
+    """
+    Test suite for Organization's courses API endpoint.
+    """
+
+    def test_api_organization_courses_read_list_anonymous(self):
```

Comment:
Ditto, Can you add all test to ensure that read operations are not allowed (POST, PUT, PATCH and DELETE). Take a look here :
https://github.com/openfun/joanie/blob/96dcd4bdc462f78b2a4a00d60c264b1954fca38e/src/backend/joanie/tests/core/test_api_course_product_relations.py